state/core_access.go

```diff
@@ -178,6 +178,7 @@ func (ca *CoreAccessor) SubmitPayForBlob(
 		ctx,
 		ca.signer,
 		ca.coreConn,
+		sdktx.BroadcastMode_BROADCAST_MODE_BLOCK,
```

Comment:
kept this the same for now as I'm not sure how you all wanted to handle this, but we can change this to sync now

Reply:
Let's do Sync please as you suggested on that issue

Reply:
originally I didn't change that as it seemed like a relatively breaking change as users would now have to have some other mechanism to know when their tx got included. 

Do you think that makes sense do that in a different PR? Keep this one just for bumping app, and then another follow up to change how light clients figure out if their tx was accepted or rejected. Happy to help out with this as I'm pretty familar with what can occur and what each response means

Reply:
> Do you think that makes sense do that in a different PR? 

Makes sense

Reply:
I think the default SubmitPFB method should be blocking and we can add an option to make it async at some point but not in this PR pls

Reply:
Block is getting deprecated, so we should migrate from it